go/github/LocalSources.qll

```diff
@@ -0,0 +1,27 @@
+import go
+
+// ========== Sources ==========
+module LocalSources {
+  abstract class Sources extends DataFlow::Node { }
+
+  // https://pkg.go.dev/os#Args
+  // class OsArgs extends Sources {
```

Comment:
We need to support os.Args as this is the main use case

Reply:
@securingdev Can you help fix this issue?

Reply:
Happy to take a crack at it @GeekMasher  - can we connect at some point tomorrow so that I know what you're looking for (and why this section of code was in, and then commented out)?